### ISSUE TITLE
训练成功启动但是不走步数，显存占用2%不到

### ISSUE THREAD

**Issue by Wpb728100** [Akegarasu/lora-scripts#563]
override steps. steps for 10 epochs is / 指定エポックまでのステップ数: 16790
running training / 学習開始
  num train images * repeats / 学習画像の数×繰り返し回数: 1679
  num reg images / 正則化画像の数: 0
  num batches per epoch / 1epochのバッチ数: 1679
  num epochs / epoch数: 10
  batch size per device / バッチサイズ: 1
  gradient accumulation steps / 勾配を合計するステップ数 = 1
  total optimization steps / 学習ステップ数: 16790
steps:   0%|                                                                                                                                                                                                                                                    | 0/16790 [00:00<?, ?it/s]
epoch 1/10
这个阶段之后，光标在闪，但是步数不能动。
